pkg/controllers/csi/metadata/models.go

```diff
@@ -24,8 +24,8 @@ type TenantConfig struct {
 	MaxFailedMountAttempts      int64  `gorm:"default:10"`
 }
 
-func (tc *TenantConfig) BeforeCreate(tx *gorm.DB) error {
-	tc.UID = uuid.NewString()
+func (tenantConfig *TenantConfig) BeforeCreate(_ *gorm.DB) error {
```

Comment:
@wepudt 
>The name of a method's receiver should be a reflection of its identity; often a one or two letter abbreviation of its type suffices (such as "c" or "cl" for "Client"). Don't use generic names such as "me", "this" or "self", identifiers typical of object-oriented languages that gives the method a special meaning. In Go, the receiver of a method is just another parameter and therefore, should be named accordingly. The name need not be as descriptive as that of a method argument, as its role is obvious and serves no documentary purpose. It can be very short as it will appear on almost every line of every method of the type; familiarity admits brevity. Be consistent, too: if you call the receiver "c" in one method, don't call it "cl" in another.

from https://go.dev/wiki/CodeReviewComments#:~:text=The%20name%20of%20a%20method's,the%20method%20a%20special%20meaning.

Reply:
fixed in 34153f4cfc53416af722d0fbf4d55583aa7c24bb